tests/lib/conftest.py

```diff
@@ -1,155 +1,68 @@
     # Create the test DB engine
     assert config.test_mode
+    assert requests.post != api_client.post
     engine = get_db_engine(
-        database_uri=config.database.sqlalchemy_database_uri,
+        database_uri=config.database.sqlalchemy_test_database_uri,
     )
 
     create_citext_extension(engine)
 
-    if not database_exists(engine.url):
-        create_database(engine.url)
-    # Create the database tables
-    Base.metadata.create_all(engine)
-
     SessionLocal = get_db_session(config, engine=engine)
-    session = SessionLocal()
-    yield session
+    the_session = SessionLocal()
 
-    session.close()
+    yield the_session
 
+    the_session.close()
     engine.dispose()
 
 
-@pytest.fixture(autouse=True, scope="session")
-def env_vars():
-    os.environ["TESTING"] = "True"
-
-
-@pytest.fixture(scope="session")
-def client():
-    """Starlette test client to use in testing API routes."""
-    app = FastAPI()
-    app.include_router(router)
-    with TestClient(app) as client:
-        yield client
-
-
-@pytest.fixture
-def fides_toml_path():
-    yield ROOT_PATH / ".fides" / "fides.toml"
-
-
-@pytest.fixture
-def oauth_client(db):
-    """Return a client for authentication purposes."""
-    client = ClientDetail(
-        hashed_secret="thisisatest",
-        salt="thisisstillatest",
-        scopes=SCOPES,
-    )
-    db.add(client)
-    db.commit()
-    db.refresh(client)
-    yield client
-    client.delete(db)
-
-
-@pytest.fixture
-def auth_header(request, oauth_client, config):
-    client_id = oauth_client.id
-
-    payload = {
-        JWE_PAYLOAD_SCOPES: request.param,
-        JWE_PAYLOAD_CLIENT_ID: client_id,
-        JWE_ISSUED_AT: datetime.now().isoformat(),
-    }
-    jwe = generate_jwe(json.dumps(payload), config.security.app_encryption_key)
-
-    return {"Authorization": "Bearer " + jwe}
-
-
-@pytest.fixture
-def user(db):
-    user = FidesUser.create(
-        db=db,
-        data={
-            "username": "test_fidesops_user",
-            "password": "TESTdcnG@wzJeu0&%3Qe2fGo7",
-        },
-    )
-    client = ClientDetail(
-        hashed_secret="thisisatest",
-        salt="thisisstillatest",
-        scopes=SCOPES,
-        user_id=user.id,
-    )
-
-    FidesUserPermissions.create(
-        db=db, data={"user_id": user.id, "scopes": [PRIVACY_REQUEST_READ]}
-    )
-
-    db.add(client)
-    db.commit()
-    db.refresh(client)
-    yield user
-    user.delete(db)
-
-
-@pytest.fixture(scope="function")
-def application_user(db, oauth_client):
-    unique_username = f"user-{uuid4()}"
-    user = FidesUser.create(
-        db=db,
-        data={
-            "username": unique_username,
-            "password": "test_password",
-            "first_name": "Test",
-            "last_name": "User",
-        },
-    )
-    oauth_client.user_id = user.id
-    oauth_client.save(db=db)
-    yield user
-    user.delete(db)
+@pytest.fixture(autouse=True)
+def clear_db_tables(db):
```

Comment:
nice that you've added this here

Reply:
I tried to add it everywhere but some of the ctl tests broke with it in place 😞